app/controllers/recipes_controller.rb

```diff
@@ -1,26 +1,22 @@
 class RecipesController < ApplicationController
-  load_and_authorize_resource :only => :index
-  
-  load_and_authorize_resource :user, :find_by => :slug, :except => [:index]
-  load_and_authorize_resource :recipe, :find_by => :url, :through => :user, :except => [:index]
+# load_and_authorize_resource :only => :index
+# load_and_authorize_resource :user, :find_by => :slug, :except => [:index]
+# load_and_authorize_resource :recipe, :find_by => :url, :through => :user, :except => [:index]
```

Comment:
Line is too long. [95/80]<br>Incorrect indentation detected (column 0 instead of 2).